subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -187,8 +187,10 @@
 #define EVENT_IFS_US            150
 /* Standard allows 2 us timing uncertainty inside the event */
 #define EVENT_IFS_MAX_US        (EVENT_IFS_US + EVENT_CLOCK_JITTER_US)
-/* Controller will layout extended adv with minimum separation */
+/* Specification defined Minimum AUX Frame Space (MAFS) */
 #define EVENT_MAFS_US           300
+/* Controller dependent MAFS minimum used to populate aux_offset */
+#define EVENT_MAFS_MIN_US       MAX(EVENT_MAFS_US, 300)
```

Comment:
To have this configurable, I believe this should be done through a Kconfig or a define in a file that is under the control of the vendor.

Reply:
A suitable name for `EVENT_MAFS_MIN_US` and corresponding Kconfig option welcome...

Reply:
How about a define `PDU_ADV_MIN_AUX_OFFSET_US` in lll_adv_pdu.h that is then (conditionally) used here to define `EVENT_MAFS_MIN_US` as `MAX(EVENT_MAFS_US, PDU_ADV_MIN_AUX_OFFSET_US)`?